libun7zip/PSPBUILD

```diff
@@ -0,0 +1,28 @@
+pkgname=libun7zip
+pkgver=19.00
+pkgrel=1
+pkgdesc="A library that provides 7-Zip (.7z) archive handling and extraction"
+arch=('mips')
+license=('Apache-2.0 license')
+url="https://github.com/bucanero/libun7zip"
+makedepends=()
+source=("git+https://github.com/bucanero/${pkgname}#commit=23e7cecd979fd893a6a7d3fcc51b2478cd860607")
+sha256sums=('SKIP')
+
+build() {
+  cd $pkgname
+  make -f Makefile.psp install
```

Comment:
This command should probably not contain install. We do not want the user's system to be modified by a PSPBUILD.

Reply:
Removed `install`, it should be fixed now.